ProjectK.Business/ExperiencePoints/ExperienceService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Logging;
+using ProjectK.Business.Characters;
+using ProjectK.Contracts;
+using System;
+using System.Threading.Tasks;
+
+namespace ProjectK.Business.ExperiencePoints
+{
+    class ExperienceService : IExperienceService
+    {
+        private readonly IExperienceRepository _repository;
+        private readonly ICharacterRepository _characterRepository;
+        private readonly ILogger _logger;
+
+        public ExperienceService(IExperienceRepository repository, ICharacterRepository characterRepository, ILogger logger)
+        {
+            _repository = repository;
+            _characterRepository = characterRepository;
+            _logger = logger;
+        }
+
+
+        public async Task<ExperienceDto> GetCharacterLevel(int charId)
+        {
+            _logger.LogInformation("Get character level: {charId}", charId);
+
+            var charLevel = await _repository.GetCharacterLevel(charId);
+
+            return charLevel?.ToDto();
+        }
+
+        public Task<ExperienceDto> GetLevelUp(int charId)
```

Comment:
This does not return anything. And possibly doesn't even compile. Do not submit PRs for code that doesn't build.

Reply:
This is why I named the PR like I did